src/cls/user/cls_user.cc

```diff
@@ -262,16 +268,20 @@ static int cls_user_remove_bucket(cls_method_context_t hctx, bufferlist *in, buf
     return ret;
   }
 
-  if (entry.user_stats_sync) {
-    dec_header_stats(&header.stats, entry);
-  }
+
+  /* if (entry.user_stats_sync) { */
```

Comment:
Yeah I'm digging into this right now, initially when I was removing without forcing the sync, the count wasn't decreased have to figure when this is set.